cmake/Abseil.cmake

```diff
@@ -53,7 +53,10 @@ macro(p4c_obtain_abseil)
         # Do not suppress warnings for Abseil library targets that are aliased.
         get_target_property(target_type ${target} TYPE)
         if (NOT ${target_type} STREQUAL "INTERFACE_LIBRARY")
-          set_target_properties(${target} PROPERTIES COMPILE_FLAGS "-Wno-error -w")
+          if (CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 14)
+            target_compile_options(${target} PUBLIC "-march=native")
```

Comment:
Maybe add a cmake warning there? This could produce surprise incompatibilities in the binary. And it might not be that far in future before someone tries to build a release binary with GCC 14.

Reply:
Also it might be good to reference the abseil issue in a comment here. Hopefully we will be able to remove this hack in future.